templates/resources/resource_policy.md.tmpl

```diff
@@ -0,0 +1,21 @@
+# {{.Type}}: {{.Name}}
+
+`{{.Name}}` provides a Resource Policy resource. The resource lets you create, edit and delete resource policies to prevent misconfigurations and reduce the need for corrective interventions in your organization.
+
+-> **NOTE**: Resource Policies are currently in Public Preview. You will need to
```

Comment:
same comment as on first file

Reply:
Changed in a8851acdb8c44933b67c874a3160f5b6938a5ba2